app/controllers/commontator/comments_controller.rb

```diff
@@ -56,10 +56,9 @@ def create
           user_practice = UserPractice.find_or_create_by(practice_id: @comment.thread.commontable_id, user: @commontator_user)
           @practice = Practice.find_by_id(@comment.thread.commontable_id)
           params[:refresh_page] = user_practice.verified_implementer != (params[:user_practice_status] == 'verified_implementer')
-
-          user_practice.update_attributes(verified_implementer: true, team_member: false) if params[:user_practice_status] == 'verified_implementer'
-          user_practice.update_attributes(verified_implementer: false, team_member: true) if params[:user_practice_status] == 'team_member'
-
+          user_practice.update_attributes(verified_implementer: true, team_member: false, other: false) if params[:user_practice_status] == 'verified_implementer'
+          user_practice.update_attributes(verified_implementer: false, team_member: true, other: false) if params[:user_practice_status] == 'team_member'
+          user_practice.update_attributes(verified_implementer: false, team_member: false, other: true) if params[:user_practice_status] == 'other'
```

Comment:
if they don't select any of the options should we default them to "other" or leave the behavior as-is?

Reply:
THis data is derived from the user_practices table... if you've never commented on a practice.. no radio buttons are selected... if you select a radio button and comment... the app will remember your "state" and pre-populate the same radio button if you make another comment.  You don't have to select an option if you don't want to.   
![image](https://user-images.githubusercontent.com/60527788/129782793-e2049e6f-7e6a-4f5b-8c1a-496609038ed3.png)